app/src/main/java/org/oppia/app/topic/play/StorySummaryAdapter.kt

```diff
@@ -38,7 +40,20 @@ class StorySummaryAdapter(
     RecyclerView.ViewHolder(binding.root) {
     internal fun bind(storySummary: StorySummary, @Suppress("UNUSED_PARAMETER") position: Int) {
       binding.setVariable(BR.storySummary, storySummary)
-      binding.root.setOnClickListener {
+
```

Comment:
Please remove empty line

Reply:
This is for clarity because next fews lines are actually a single line that's why.